src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/Microsoft.VisualStudio.Web.CodeGeneration.Utils.csproj

```diff
@@ -26,13 +26,12 @@
     <PackageReference Include="NuGet.Frameworks" Version="$(NuGetFrameworksVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
-    <PackageReference Include="System.AppContext" Version="$(CoreFxVersion)" />
-    <PackageReference Include="System.Security.Cryptography.Algorithms" Version="$(CoreFxVersion)" />
-    <PackageReference Include="System.Threading.Tasks.Parallel" Version="$(CoreFxVersion)" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="System.Runtime.Loader" Version="$(CoreFxVersion)" />
```

Comment:
Cannot target `netcoreapp2.0` here as this assembly is lower down the dependency chain in scaffolding. 
It would mean all other packages would need to be converted to target `netcoreapp2.0` and `net461`